pallets/accounts/src/dummy_data.rs

```diff
@@ -14,7 +14,35 @@ use sp_core::{
 fn get_ethereum_chain_id() -> u64 {
 	use crate::mock::Test;
 	use frame_support::traits::TypedGet;
-	<Test as crate::Config>::EthChainId::get()
+	<Test as Config>::EthChainId::get()
+}
+
+#[cfg(test)]
+pub fn get_ticket_asset_id() -> u32 {
+	use crate::mock::Test;
+	use frame_support::traits::TypedGet;
+	<Test as Config>::TicketsAssetId::get()
```

Comment:
What does it test? I only see a single call.

Shouldn't there be some `assert` with expected values? As well as failure cases.

Reply:
This is a utility to retrieve the TicketsAssetId, which is defined into the runtime Config
```rust
parameter_types! {
	pub const TicketsAssetId: u64 = 1337;
}
```
It does not test anything, but yeah good idea I should assert the returned value.

Reply:
Done I added tests for all this config.